src/main/webapp/index.html

```diff
@@ -12,18 +12,14 @@
 
   <body>
     <div class="jumbotron">
-        <h1>SDK Firebase Release Dashboard</h1>  
+        <h1 class="dashboard">SDK Firebase Release Dashboard</h1>  
     </div>  
 
     <div class="container-fluid">
       <div class="row content justify-content-center">
         <div class="col-sm-2 sidenav">
-          <h1>Favorite SDKs</h1>
-                <ul class="nav nav-pills nav-stacked">
-                    <li><a href="#section1">Firebase Storage Api</a></li>
-                    <li><a href="#section2">Measurement</a></li>
-                    <li><a href="#section3">Tagmanager</a></li>
-                    <li><a href="#section3">Firebase Common</a></li>
+          <h1 >Favorite SDKs</h1>
+                <ul id="favorites">
```

Comment:
Did you want to call any method(getFavoriteSdks?) here? It looks like you have made this an empty list, not sure if I am missing anything.

Reply:
So for this method the user id was not specified so I could not fetch any dummy data, but I tested the method and it worked.